lib/twitter.js

```diff
@@ -32,26 +33,42 @@ function Twitter (options) {
       headers: {
         'Accept': '*/*',
         'Connection': 'close',
-        'User-Agent': 'node-twitter/' + VERSION
+        'User-Agent': 'node-twitter/' + VERSION,
```

Comment:
Remove the trailing slash here.  Its valid, but its not consistent with the rest of the codebase.